javascript/computer_science/project_linked_lists.md

```diff
@@ -1,29 +1,27 @@
 ### Introduction
 
-In Computer Science one of the most basic and fundamental data structures is the
+In Computer Science, one of the most basic and fundamental data structures is the
 linked list, which functions similarly to an array. The principal benefit of a linked
 list over a conventional array is that the list elements can easily be inserted or
-removed without reallocation of any other elements.
+removed without the reallocation of any other elements.
 
-In some programming languages the size of an array is a concern and one of the ways
-to overcome that problem and allow dynamically allocated data is using linked lists.
+In some programming languages, the size of an array is a concern. As such, linked lists are one way for programmers to overcome this constraint while allowing dynamically allocated data in the process.
 
 Luckily in **Javascript** arrays aren't limited to a certain size, so you don't have to think
 about overcoming that limitation.
 
 So if array size is not a limitation in Javascript, are linked lists really necessary?
-The short answer to that is _no_; however, it's the simplest of the dynamic data
-structures and it will give you a solid foundation, so you can understand more
-complex data structures like graphs and binary trees with more ease.
+The short answer is _no_; however, it's the simplest of the dynamic data
+structures and will give you a solid foundation to more easily understand
+complex data structures like graphs and binary trees.
 
 ### Structure of a Linked List
 
-A _linked list_ is a linear collection of data elements called nodes that "point"
-to the next node by means of a pointer.
+A _linked list_ is a linear collection of data elements called nodes which utilize pointers to denote the next node in the list.
```

Comment:
While we used the term "pointer" before, it is a technical term that would be familiar to people who have used languages such as C, but would likely not be familiar to many people going through the curriculum. Can you edit this to make things a bit clearer as to what a pointer is if you're going to use the term?